meta/template/jit.lua

```diff
@@ -5,29 +5,39 @@
 ---@class jitlib
 ---@field version     string
 ---@field version_num number
----@field os          string
----@field arch        string
+---@field os          'Windows'|'Linux'|'OSX'|'BSD'|'POSIX'|'Other'
+---@field arch        'x86'|'x64'|'arm'|'arm64'|'arm64be'|'ppc'|'ppc64'|'ppc64le'|'mips'|'mipsel'|'mips64'|'mips64el'|string
```

Comment:
Is there a reason you left `|string` at the end of this union?

Reply:
Some branches support other architectures

Reply:
That's a valid reason. Thanks for the explanation.